tests/rhui3_tests/test_entitlements.py

```diff
@@ -42,12 +43,36 @@ def test_04_upload_rh_certificate():
     list = RHUIManagerEntitlements.upload_rh_certificate(connection)
     nose.tools.assert_not_equal(len(list), 0)
 
-def test_02_list_rh_entitlements():
+def test_05_list_rh_entitlements():
     '''
        list Red Hat content certificate entitlements
     '''
     entitlements = RHUIManagerEntitlements.list_rh_entitlements(connection)
     nose.tools.eq_(isinstance(entitlements, list), True)
 
+def test_06_add_custom_repo():
+    '''
+       add a custom repo to protect by a client entitlement certificate
+    '''
+    Expect.enter(connection, "home")
+    Expect.expect(connection, ".*rhui \(" + "home" + "\) =>")
+    RHUIManagerRepo.add_custom_repo(connection, "custom-enttest", "", "", "1", "y")
+
+def test_07_list_custom_entitlements():
+    '''
+       list custom content certificate entitlements, expect one
+    '''
+    list = RHUIManagerEntitlements.list_custom_entitlements(connection)
+    nose.tools.assert_equal(len(list), 1)
+
+def test_08_remove_custom_repo():
+    '''
+       remove the custom repo
+    '''
+    Expect.enter(connection, "home")
+    Expect.expect(connection, ".*rhui \(" + "home" + "\) =>")
+    RHUIManagerRepo.delete_repo(connection, ["custom-enttest"])
+    nose.tools.assert_equal(RHUIManagerRepo.list(connection), [])
```

Comment:
What about checking that list_custom_entitlements() will return [] after removing the repo?